tools/SetupFlow/DevHome.SetupFlow/Views/QuickstartPlaygroundView.xaml

```diff
@@ -49,6 +50,82 @@
                 <ResourceDictionary Source="ms-appx:///DevHome.SetupFlow/Styles/SetupFlowStyles.xaml" />
                 <ResourceDictionary Source="ms-appx:///DevHome.SetupFlow/Styles/QuickstartStyles.xaml" />
             </ResourceDictionary.MergedDictionaries>
+            <DataTemplate x:Key="ChatResponseMessageTemplate" x:DataType="viewmodels:ChatStyleMessage">
+                <Grid Height="Auto"
+			Margin="4"
+			HorizontalAlignment="Left">
+                    <Grid.RowDefinitions>
+                        <RowDefinition Height="Auto" />
+                        <RowDefinition Height="Auto" />
+                    </Grid.RowDefinitions>
+                    <StackPanel Grid.Row="1"
+				Padding="16,8,16,8"
+				CornerRadius="16">
+                        <StackPanel.Background>
+                            <SolidColorBrush Color="WhiteSmoke" />
+                        </StackPanel.Background>
+                        <Border Padding="8,4,8,4"
+									CornerRadius="4">
+                            <Border.Background>
+                                <SolidColorBrush Opacity="0.2"
+							Color="WhiteSmoke" />

```

Comment:
Please us `ThemeResource` for `opacity` and `Color` so the UI can change to events.  For example, switching from dark mode to light mode.